### PR TITLE
add `cupy.random.hypergeometric`

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

File: tests/cupy_tests/random_tests/test_generator.py [hunk 1]
```diff
@@ -202,6 +202,21 @@ def test_geometric(self):
...
+    {'ngood': 1, 'nbad': 1, 'nsample': 1},
+    {'ngood': 1, 'nbad': 1, 'nsample': 2},
+)
+@testing.gpu
+@testing.fix_random()
+class TestHypergeometric(RandomGeneratorTestCase):
+
+    target_method = 'hypergeometric'
+
+    def test_geometric(self):
```
Comment (toslunar) [inline tests/cupy_tests/random_tests/test_generator.py]: Could you fix the name of the test method?

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
Could you resolve the conflict?  (`loggam` is added to `_kernels` in #1418)

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
Could you check the test failures with `cupy-py2`?

**Comment by YoshikawaMasashi** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failure is irrelevant.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**